client/src/main/java/com/orientechnologies/orient/client/remote/OServerAdmin.java

```diff
@@ -101,9 +100,8 @@ public Void execute(OChannelBinaryAsynchClient network) throws IOException {
           String password = iUserPassword;
 
           OCredentialInterceptor ci = OSecurityManager.instance().newCredentialInterceptor();

```

Comment:
Since uname and passwd aren't used, I'll prefer to extract a method "setupCredentialIntercerptor"